docs/components/topnav/index.html

```diff
@@ -0,0 +1,98 @@
+---
+title: Top Navigation
+minver: 1.1.0
+also:
+  components/topnav/test.html: Full Page Demo - Top Navigation
+  elements/hx-navigation-control: <hx-navigation-control>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    The {{page.title}} provides access to platform-level services and aggregate information
+    across a user's accounts at Rackspace.
+
+  </p>
+{% endblock %}
+
+  {% block content %}
+
+  <section>
+    <header>
+      <h2 id="icon-badge">Nav Menu</h2>
+    </header>
+
+    <div id="vue-topNavDemo">
+      <div id="hxTopnav">
+        <a class="img-left brand-logo" href="#"> <img src="images/helix-logo.svg" alt="Logo" /></a>
+        <div class="productMenu">
+            <hx-disclosure id="topnav-productMenu" aria-controls="topnavProductDropdown" role="button" aria-expanded="true"
+              class="disabled">
+              <span>Select a Product</span>
+              <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+            </hx-disclosure>
+            <hx-menu id="topnavProductDropdown">
+              <div id="sectionOne">
+                <hx-menuitem role="menuitem">Product alpha</hx-menuitem>
+                <hx-menuitem role="menuitem">Product Beta Services</hx-menuitem>
+                <hx-menuitem role="menuitem">Product Charlie Service Offering</hx-menuitem>
+              </div>
+            </hx-menu>
+          </div>
+        <div id="right-menu">
+          <div>
+            <a href="#" @click="selectedTag(tab)" :current-tab="currentTab">
+              <hx-icon type="bell"></hx-icon>
+              <p>Notifications</p>
+            </a>
+
+            <a href="#" class="selected">
+              <hx-icon type="ticketing"></hx-icon>
+              <p>Tickets</p>
+            </a>
+
+            <a href="#">
+              <hx-icon type="support"></hx-icon>
+              <p>Support</p>
+            </a>
+
+            <a href="#" class="disabled" id="billing">
+              <hx-icon type="billing"></hx-icon>
+              <p>Billing</p>
+              <hx-tooltip for="billing" position="bottom-center">
+                You do not have access to this area. Contact an account admin in your organization to request access to
+                this item.
+              </hx-tooltip>
+            </a>
+
+            <a href="#">
+              <hx-icon type="account"></hx-icon>
+              <p>Account</p>
+            </a>
+          </div>
+          <div class="straightLine"></div>
+          <div>
+            <hx-disclosure id="topnav-menu" aria-controls="topnavMenuId" role="button" aria-expanded="true"
+              class="disabled">
+              <span>Jane User</span>
+              <hx-icon class="hxPrimary" type="angle-down"></hx-icon>
+            </hx-disclosure>
+            <hx-menu id="topnavMenuId">
```

Comment:
Please add the **Account** back to the user menu:

<img width="253" alt="Screen Shot 2020-10-06 at 11 44 08 AM" src="https://user-images.githubusercontent.com/10120600/95264524-17a4f280-07f5-11eb-9ecf-9f8c4d819f4a.png">